app/views/miq_request/_prov_field.html.haml

```diff
@@ -290,26 +289,20 @@
             = field_hash[:notes]
       - elsif [:tag_ids, :vm_tags].include?(field)
         -# tree control for tags fields
-        .col-md-8
+        .col-md-10
           #all_tags_treebox.treeview-pf-hover.treeview-pf-select{:style => "color:#000; overflow: hidden;"}
```

Comment:
We should probably remove these tree classes and style overrides since this is no longer a tree.

Reply:
Also, the div is actually empty, and the code setting `check` is not used anymore, so you can just remove lines 293 to 299